### PR TITLE
Updated features.yaml for Serverless Computing Functions

### PR DESCRIPTION
**Pull request by kazmik23**

### CODE REVIEW

File: services/compute/serverless-computing/features.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+features:
+  - id: CCC.SvlsComp.F01  # Function Execution
+    title: Function Execution
+    description: |
+      Supports execution of code functions in response to events without the need to manage underlying server infrastructure.
+    references: # Temporary for validation purposes
+      AWS: https://docs.aws.amazon.com/lambda/latest/dg/invocation-types.html
+      GCP: https://cloud.google.com/functions/docs/concepts/exec
+      Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:23]:
```suggestion
    # references: #
    #  AWS: https://docs.aws.amazon.com/lambda/latest/dg/invocation-types.html
    #  GCP: https://cloud.google.com/functions/docs/concepts/exec
    #  Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview
```

File: services/compute/serverless-computing/features.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - CCC.F11  # Backup
+  - CCC.F12  # Restore
+  - CCC.F14  # API Access
+  - CCC.F19  # On-Demand Scaling
+
+features:
+  - id: CCC.SvlsComp.F01  # Function Execution
+    title: Function Execution
+    description: |
+      Supports execution of code functions in response to events without the need to manage underlying server infrastructure.
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:19]:
```suggestion
      Supports execution of code functions in response
      to events without the need to manage underlying
      server infrastructure.
```

File: services/compute/serverless-computing/features.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    # references: #
+    #  AWS: https://docs.aws.amazon.com/lambda/latest/dg/invocation-types.html
+    #  GCP: https://cloud.google.com/functions/docs/concepts/exec
+    #  Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview
+
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:24]:
```suggestion

```

File: services/compute/serverless-computing/features.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    # references: #
+    #  AWS: https://docs.aws.amazon.com/lambda/latest/dg/invocation-types.html
+    #  GCP: https://cloud.google.com/functions/docs/concepts/exec
+    #  Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview
+   
+
+  - id: CCC.SvlsComp.F02  # Event Triggers
+    title: Event Triggers
+    description: |
+      Supports triggering functions in response to events from various sources such as HTTP requests, cloud storage changes, messaging services, and scheduled events.
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:29]:
```suggestion
      Supports triggering functions in response to events
      from various sources such as HTTP requests, cloud
      storage changes, messaging services, and scheduled
      events.
```

File: services/compute/serverless-computing/features.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    references:
+      AWS: https://docs.aws.amazon.com/lambda/latest/dg/lambda-services.html
+      GCP: https://cloud.google.com/functions/docs/calling/triggers
+      Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:33]:
```suggestion
    # references:
    #   AWS: https://docs.aws.amazon.com/lambda/latest/dg/lambda-services.html
    #   GCP: https://cloud.google.com/functions/docs/calling/triggers
    #   Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings
```

File: services/compute/serverless-computing/features.yaml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:34]:
```suggestion

```

File: services/compute/serverless-computing/features.yaml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    references:
+      AWS: https://docs.aws.amazon.com/lambda/latest/dg/lambda-services.html
+      GCP: https://cloud.google.com/functions/docs/calling/triggers
+      Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings
+ 
+
+  - id: CCC.SvlsComp.F04  # VPC Connectivity
+    title: VPC Connectivity
+    description: |
+      Supports connecting functions to a Virtual Private Cloud (VPC) network using VPC connectors, enabling secure access to resources in a private network.
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:39]:
```suggestion
      Supports connecting functions to a Virtual Private
      Cloud (VPC) network using VPC connectors, enabling
      secure access to resources in a private network.
```

File: services/compute/serverless-computing/features.yaml [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    references:
+      AWS: https://docs.aws.amazon.com/lambda/latest/dg/configuration-vpc.html
+      GCP: https://cloud.google.com/functions/docs/connecting-vpc
+      Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-networking-options
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:43]:
```suggestion
    # references:
    #   AWS: https://docs.aws.amazon.com/lambda/latest/dg/configuration-vpc.html
    #   GCP: https://cloud.google.com/functions/docs/connecting-vpc
    #   Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-networking-options
```

File: services/compute/serverless-computing/features.yaml [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+
+  - id: CCC.SvlsComp.F05  # Environment Variables
+    title: Environment Variables
+    description: |
+      Allows setting environment variables for functions to store configuration settings and operational parameters.
+    references:
+      AWS: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
+      GCP: https://cloud.google.com/functions/docs/configuring/env-vars
+      Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-how-to-use-azure-function-app-settings
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:53]:
```suggestion
    # references:
    #   AWS: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
    #   GCP: https://cloud.google.com/functions/docs/configuring/env-vars
    #   Azure: https://learn.microsoft.com/en-us/azure/azure-functions/functions-how-to-use-azure-function-app-settings
```

File: services/compute/serverless-computing/features.yaml [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:54]:
```suggestion
```

File: services/compute/serverless-computing/features.yaml [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+    title: Event Driven Execution
+    description: |
+      Supports execution of code functions in response to events
+      without the need to manage underlying server infrastructure.
+
+  - id: CCC.SvlsComp.F02
+    title: Event Triggers
+    description: |
+      Ability to configure event triggers for functions such as
+      HTTP requests, cloud storage changes, messaging services,
```
Comment (eddie-knight) [inline services/compute/serverless-computing/features.yaml:24]:
```suggestion
      HTTP requests, cloud storage changes, messaging services,
```

**Comment by smendis-scottlogic** [comment]
@eddie-knight sorry for the confusions. I did not know you was working on this PR. I have added my changes. Please review if possible. I suggest changing the service code to "Func" or "Fns" and folder name to "functions" I will do it in another PR after this gets merged.

**Pull request merged**